source/uwp/SharedRenderer/lib/AdaptiveCardRendererComponent.cpp

```diff
@@ -78,6 +78,16 @@ namespace winrt::AdaptiveCards::Rendering::Xaml_Rendering::implementation
         GetHostConfig()->OverflowMaxActions = overflowMaxActions;
     }
 
+    bool AdaptiveCardRenderer::AddSelectActionMargin()
+    {
+        return GetHostConfig()->AddSelectActionMargin;
+    }
+
+    void AdaptiveCardRenderer::AddSelectActionMargin(bool addSelectActionMargin)
```

Comment:
do we need this method since the value is read from host config?

Reply:
Yes, I believe so. The usage is like this:

`_renderer.AddSelectActionMargin = false;`

I'm actually not sure why we store the values in the hostConfig for this type of property. Maybe because we don't have access to renderer properties when we construct the objects? @paulcam206

Reply:
Do we maintain parity of HostConfig capabilities between the platforms today? Because if we do, adding this property to HostConfig would break that. @paulcam206

Reply:
@jwoo-msft Do you have any additional context here?

From my understanding, this doesn't maintain parity; however, it isn't an issue for the other platfroms because it wasn't added to the _shared_ host config

Reply:
Somewhat related - do we have a plan for decoupling from the mobile renderers?

Reply:
we don't put much emphasis on parities across platforms anymore. We only support UWP/WinRT and Type Script renderers, and we have allowed them to diverge.

Reply:
> Somewhat related - do we have a plan for decoupling from the mobile renderers?

the coupling doesn't exist today. Mobile renderers have their own copy of shared code.